types/mirrorx/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/mirrorjs/mirror
 // Definitions by: Aaronphy <https://github.com/aaronphy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.0
```

Comment:
Note that increasing the TS version prevents consumers on older versions from using these type definitions.

Reply:
Yes, and this will be the case up to TS 3.0.

It has to be bumped because react-redux itself was bumped, and http://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#generic-rest-parameters is needed for this to work correctly.